Frontend/src/pages/dashboard/components/Farmer/StatisticsDashboard.jsx

```
@@ -7,88 +8,141 @@ import {
+        <CardHeader>
+          <CardTitle className="text-xl font-bold">Monthly Sales</CardTitle>
+          <CardDescription>Revenue for the last 3 months</CardDescription>
+        </CardHeader>
+        <CardContent>
+          <div className="h-[300px] flex items-end justify-between gap-2 pt-4">
+            {stats.monthlySales.map((item) => {
+              const maxAmount = Math.max(...stats.monthlySales.map((sale) => sale.amount), 1000); // Avoid division by zero
+              const height = (item.amount / maxAmount) * 100; // Scale dynamically
+              return (
+                <div key={item.month} className="flex flex-col items-center gap-2">
+                  <div className="w-16 bg-primary rounded-t-md" style={{ height: `${height}%` }}></div>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)